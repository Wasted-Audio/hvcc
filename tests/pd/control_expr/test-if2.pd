#N canvas 827 239 734 565 12;
#X obj 55 65 loadbang;
#X obj 55 325 print;
#X msg 55 175 2.3;
#X obj 55 140 t b b b;
#X obj 55 97 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X obj 241 260 expr if($f1 > 0 \, $f1 * 10 \, $f1 - 1);
#X obj 467 260 expr if($f1 > 10 \, 10 \, if($f1 < -10 \, -10 \, $f1));
#X obj 55 260 expr if($f1 >= 0 \, 1 \, -1);
#X msg 192 174 12;
#X msg 109 175 -3.1;
#X connect 0 0 4 0;
#X connect 2 0 5 0;
#X connect 2 0 6 0;
#X connect 2 0 7 0;
#X connect 3 0 2 0;
#X connect 3 1 9 0;
#X connect 3 2 8 0;
#X connect 4 0 3 0;
#X connect 5 0 1 0;
#X connect 6 0 1 0;
#X connect 7 0 1 0;
#X connect 8 0 5 0;
#X connect 8 0 6 0;
#X connect 8 0 7 0;
#X connect 9 0 5 0;
#X connect 9 0 6 0;
#X connect 9 0 7 0;
