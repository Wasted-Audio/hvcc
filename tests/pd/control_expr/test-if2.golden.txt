[@ 0.000] print: 120
[@ 0.000] print: 10
[@ 0.000] print: 1
[@ 0.000] print: -4.1
[@ 0.000] print: -3.1
[@ 0.000] print: -1
[@ 0.000] print: 23
[@ 0.000] print: 2.3
[@ 0.000] print: 1
